site/docs/flink.md

```diff
@@ -312,17 +312,47 @@ INSERT OVERWRITE hive_catalog.default.sample PARTITION(data='a') SELECT 6;
 For a partitioned iceberg table, when all the partition columns are set a value in `PARTITION` clause, it is inserting into a static partition, otherwise if partial partition columns (prefix part of all partition columns) are set a value in `PARTITION` clause, it is writing the query result into a dynamic partition.
 For an unpartitioned iceberg table, its data will be completely overwritten by `INSERT OVERWRITE`.
 
-## Reading with DataStream
+## Iceberg Operation with DataStream API
+### Load Iceberg Catalog
```

Comment:
Here I think we should introduce the background why we introduce the `CatalogLoader` & `TableLoader`:  We flink operator want to access the iceberg table while Catalog & Table are not serializable because they depends on some resources that could not be serializable (such as Connection). So we have to introduce the `Loader` to maintain the configurations which are required to initialize the `Catalog`  and `Table`. 

Also we'd better to list the general `CatalogLoader`s and explain what do they mean: 

a. `HiveCatalogLoader` ; 
b. `HadoopCatalogLoader` ; 
c. `CustomCatalogLoader`. 

Ditto for `TableLoader`.